workflow_settings.yaml

```
@@ -2,4 +2,8 @@ defaultProject: httparchive
 defaultLocation: US
 defaultDataset: all
 defaultAssertionDataset: dataform_assertions
-dataformCoreVersion: 3.0.0
+
+vars:
+  placeholder: value  # keeping it to avoid 'empty vars' error
+  env_name: dev  # MUST be commented in main branch, enables processing sampled data
```

To comment out before merging:
```suggestion
  # env_name: dev  # MUST be commented in main branch, enables processing sampled data
```